src/routers/mailingList/validator.ts

```diff
@@ -0,0 +1,19 @@
+import { check } from 'express-validator';
+
+const validator = (route: string) => {
+  switch (route) {
+    case '/':
+      return [
+        check('email').normalizeEmail().isEmail(),
+        check('interestedEvents').isArray(),
+        check('interestedEvents.*').notEmpty(),
+        check('otherFeedback').isString(),
+      ];
+    default:
+      throw new Error(
+        `Validator for relative path "${route}" on mailingList router does not exist.`,
+      );
+  }
+};
+
+export default validator;
```

Comment:
I did a little bit more than "out of the box". I'm thinking we can have one validator function for each router like this that returns the correct validation checks based on the relative route. In this case we only have one route 😅  but this way it scales and stays clean when we have multiple.

I would love to have the route param type be narrowed to only the routes that are defined instead of `string` but not sure it's worth it. We'd have to do something like:
```
import { MAILING_LIST_ROUTES as ROUTES } from './constants';

router.post(ROUTES.ROOT, validator(ROUTES.ROOT), validate, ...);
```
and we'd have to define all the relative routes for each router. What do you think?